src/createServer.js

```diff
@@ -1,3 +1,62 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => http.createServer((req, res) => {
+  const url = new URL(req.url, `http://${req.headers.host}`);
+  const parts = url.pathname
+    .slice(1)
+    .split('/')
+    .filter(part => part);
+  const query = Object.fromEntries(url.searchParams.entries());
+  const errors = [];
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  if (!parts.length || parts.length > 1) {
+    errors.push({
+      message: 'Text to convert is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
Do not rely on the unknown string, make constants for this

Reply:
@AlinaMatuschak Fixed